tasks/insights-client.yml

```diff
@@ -10,15 +10,41 @@
   when:
     - rhc_state | d("present") != "reconnect"
 
+- name: Add insights tags
+  copy:
+    dest: /etc/insights-client/tags.yml
+    content: "{{ rhc_insights.tags | to_nice_yaml }}"
```

Comment:
If you need to, you can change the formatting and indentation of the generated yaml - see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#formatting-data-yaml-and-json and https://jinja.palletsprojects.com/en/2.9.x/templates/#indent

Reply:
I think is not needed to change to much the formatting, it is a simple yaml file.